requirements.txt

```diff
@@ -0,0 +1,3 @@
+Django~=3.2.10
+pylint
+python-dotenv
```

Comment:
Was this in the tutorial? It's probably not really needed in the initial skeleton.

FWIW, I was going to ask you to integrate https://django-environ.rtfd.io later on anyway as it's a more natural/complete/integrated choice.

Note though, if you want to have it integrated right away, you'd need to go full-on with the settings setup and including the readme guide. Plus it'd need a dummy `.env.example` committed that would demonstrate the env vars needed to be provided for the initial dev env config.

Reply:
@Martolivna so one of the points I was making earlier was linking the changes if they are made. But sometimes, the context requires questions fully answered.
Currently, this thread appears to be ignored, which is probably not the case. In order to show that it wasn't ignored, it's recommended to explicitly state how it's been addressed or continue the discussion otherwise.
By not doing so, you shift the burden of double-verification if something changed and cross-checking this thread with the recent commits to the reviewer / thread starter. This makes it harder / longer to make reviews, as the reviewer has to match pieces of disconnected context together to understand the whole picture. This adds to the mental overhead and may result in things being lost/missed.
I recommend going through all the threads and discussions and adding summaries / providing that context while you work on the PR since you already have this context at such times. Recording it helps both the reviewer and yourself to stay on top of what's happening and what the overall status is. It'll also help the future explorers discover why things happened the way they did, when they are digging for unobvious motivations in the future.

Reply:
For now, I removed the dotenv module and temporarily used a non-secure secret key hardcoded directly in the code. Later, I am planning to review the django-environ tool and integrate it.

Reply:
Brilliant! I'm marking this thread as resolved.